app/main.py

```diff
@@ -16,4 +16,11 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+vars_dict = locals().copy()
+sorted_variables = {"mutable": [], "immutable": []}
+mutable_types = [list, set, dict]
+for key, value in vars_dict.items():
+    if not key.startswith("__"):
+        category = "mutable" if type(vars_dict[key]) in mutable_types \
```

Comment:
Looks great, but avoid using linebreaks, you can wrap the ternary operator using (  ), and format it as you wish to.